src/corecel/sys/Device.cc

```diff
@@ -192,6 +192,13 @@ Device::Device(int id)
 
     // Save for possible block size initialization
     max_threads_per_block = props.maxThreadsPerBlock;
+
+    // TODO: Configurable threshold
```

Comment:
This would be a good candidate for an environment variable: see e.g. VecGeomParams:
```
        if (std::string var = celeritas::getenv("CUDA_STACK_SIZE");
            !var.empty())
        {
            int stack_size = std::stoi(var);
            CELER_VALIDATE(stack_size > 0,
                           << "invalid CUDA_STACK_SIZE=" << stack_size
                           << " (must be positive)");
            set_cuda_stack_size(stack_size);
        }
```